sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncClient.java

```diff
@@ -397,12 +376,13 @@ boolean isContentResponseOnWriteEnabled() {
      */
     CosmosPagedFlux<CosmosDatabaseProperties> readAllDatabases(CosmosQueryRequestOptions options) {
```

Comment:
Yes that is correct , Mono<CosmosDatabaseResponse> createDatabase(String id, int throughput) is not getting used. But I am not sure about the dev intention behind this. And this is existing in master code too, so keeping it out of Tracer scope

Reply:
it was there in master too , however delete it now

Reply:
We need to keep those APIs @samvaity  - because they will be introduced later. 
We intentionally made them package-private. They will be introduced in upcoming versions once we have the correct "read feed" implementation.  
@simplynaveen20 - please introduce it back.

Reply:
I think Sameekha was taking about createDatabase on L348 , it is not needed . so we good here